app/Http/Controllers/Api/V1/Integration/BrightcoveAPISettingsController.php

```diff
@@ -36,10 +36,10 @@ public function __construct(BrightcoveAPISettingRepository $brightcoveAPISetting
     /**
      * Get All Brightcove API Settings for listing.
```

Comment:
Add new line between title, description params and return
as mentioned in https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api